app/web/features/search/MapWrapper.tsx

```diff
@@ -56,8 +56,8 @@ const useStyles = makeStyles((theme) => ({
 
 interface mapWrapperProps {
   selectedResult:
-  | Pick<User.AsObject, "username" | "userId" | "lng" | "lat">
-  | undefined;
+    | Pick<User.AsObject, "username" | "userId" | "lng" | "lat">
+    | undefined;
```

Comment:
Will the `selectedResult` be possibly totally undefined or just an empty object? Or are some of the object keys undefined?

Reply:
hmm yap it's undefined by default, until a user selects a result :)